### DISCUSSION TITLE
Actividad 6

### DISCUSSION THREAD

**Discussion by jchernandez22** [JavierBZ/Tarea3-IIC3697#31] [Q&A]
Hola!
En el enunciado de la actividad 6 nos piden indicar los hiperparámetros pero no habla sobre indicar a qué se debe esta elección, es innecesario que expliquemos el porqué elegimos esos valores?

**Comment by JavierBZ** [comment]
Si son valores que no son los mismos que la implementación del paper de DQN deben explicar porque los cambiaron, si no no es necesario. Se asume que los pusieron basándose en el paper.
